client-js/main/client/firebase-subscription-service.js

```diff
@@ -84,6 +84,33 @@ export class FirebaseSubscriptionService {
     }
   }
 
+  /**
+   * Immediately cancels all active subscriptions previously created through this service.
+   */
+  cancelAllSubscriptions() {
+    const activeSubscriptions = this._subscriptions.filter(s => !s.closed);
+    if (activeSubscriptions.length > 0) {
+      const subscriptionMessages = activeSubscriptions.map(s => s.internal())
+      this._endpoint.cancelAll(subscriptionMessages);
+      activeSubscriptions.forEach(s => {
+        s.unsubscribe() /* Calling RxJS's `unsubscribe` to stop propagating the updates. */
+        this._removeSubscription(s)
+      })
+    }
+  }
+
+  /**
+   * Immediately cancels the given subscription, including cancelling it on the server-side.
+   *
+   * @param {SubscriptionObject} subscription the subscription to cancel
+   */
+  cancelSubscription(subscription) {
+    this._endpoint.cancelSubscription(subscription)
+        .then(() => {
+          this._removeSubscription(subscription)
```

Comment:
Here `subscription` is a Protobuf message, `_removeSubscription` expects something derived from RxJS `Subscription`.  I guess we should rework `_removeSubscription` method so it could remove subscriptions by ID.

Reply:
@YegorUdovchenko 
That's an interesting point. Perhaps, you are right. 

However, I don't understand how [this thing](https://github.com/SpineEventEngine/web/blob/v1/client-js/main/client/firebase-subscription-service.js#L143) worked previously. It uses `subscription.internal()` objects, which AFAIU are Protobuf messages.

Reply:
I don't think it actually works.

Reply:
@YegorUdovchenko 
I have reworked this removal thing to take any kind of `Subscription`, and operate by its ID. I think, that's the safest approach for now.

Reply:
@armiol 
Looks ok.